spec/factories/notereactions.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :noteReaction do
```

Comment:
```suggestion
  factory :note_reaction do
```

… also, the suggest the filename be `note_reactions.rb`.